Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos

```diff
@@ -1846,7 +1846,7 @@ convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.VoltageS
 convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.VoltageSensor","arg_y","arg_v")
 convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "y", "i")
 convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "abs_y", "abs_i")
-convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "arg_y", "arg_i")
+convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "arg_y", "arg_v")
```

Comment:
That probably is a bug that should be reverted.

Reply:
Ah, you mean a bug in the CurrentSensor, right? Yes, that is likely correct when I spend two seconds thinking of it.

Reply:
Yes, was fixed by #3488 and should be reverted here.

Reply:
Ok, done!

Reply:
I simply removed these two back and forth commits to get a clean history when merging.